backend-rust/.env

```diff
@@ -0,0 +1,2 @@
+# Postgres
+DATABASE_URL=postgres://postgres:example@localhost/ccd-scan
```

Comment:
We should use [the "offline" database checking that sqlx supports](https://github.com/launchbadge/sqlx/blob/main/sqlx-cli/README.md#enable-building-in-offline-mode-with-query) so that you don't need a database to build.